codemod/output/AuthorizationCustom.ts

```diff
@@ -0,0 +1,23 @@
+import { AuthorizationMethod } from "./AuthorizationMethod";
+
+class AuthorizationCustom extends AuthorizationMethod {
```

Comment:
You should export the classes, I took this file as an example but it is valid everywhere else.

Reply:
Hi @mtltechtemp! 

Indeed, the reason why I did this was to make it easier to scope the classes in their own file through the automatized tool, so then I wouldn't have to fiddle with the methods being defined extending the prototype.

The main goal of this PR was to improve the maintainability of the project by scoping the classes in their own files, and without changing too much of its structure, mainly to prevent the appearance of bugs.

Reply:
Ok i agree let's split this in another PR.